ml-notebook/environment.yml

```diff
@@ -5,5 +5,6 @@ channels:
  - conda-forge
 dependencies:
  - cudatoolkit=10
- - tensorflow>=2
- - jax
+ - tensorflow=2.7.0
+ - jax=0.3.0
+ - jaxlib=0.1.75
```

Comment:
pinning these all explicitly for now to versions we currently have to unblock and merge. If someone wants to revisit this in a separate PR feel free!